autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicSelect.tsx

```diff
@@ -21,6 +24,34 @@ const BasicSelect: React.FC<BasicSelectProps> = ({ id }) => {
     (state) => state.courseCards[id].hasAsynchronous || false,
   );
 
+  // generates available filter options
+  const filterOptions = (
+    <>
+      <BasicCheckbox id={id} value="includeFull" label="Include Full Sections" />
+      { hasHonors
+        ? <BasicOptionRow id={id} value="honors" label="Honors" />
+        : null }
+      { hasRemote
+        ? <BasicOptionRow id={id} value="remote" label="Remote" />
+        : null }
+      { hasAsynchronous
+        ? <BasicOptionRow id={id} value="asynchronous" label="No Meeting Times" />
+        : null }
+    </>
+  );
+
+  // if no filter options generates message to alert user
+  const noOptionsMessage = (
+    <>
+      {(!hasHonors && !hasRemote && !hasAsynchronous)
+        ? (
+          <Typography className={styles.placeholderText} color="textSecondary">
+            There are no honors or remote sections for this class
+          </Typography>
+        ) : null}
+    </>
+  );
```

Comment:
```suggestion
  const noOptionsMessage = hasHonors || hasRemote || hasAsynchronous ? null : (
    <Typography className={styles.placeholderText} color="textSecondary">
      There are no honors or remote sections for this class
    </Typography>
  );
```
`null` JSX elements render nothing, so moving the ternary to the outside makes it more clear what this is doing